### PR TITLE
small edits on readme, added new photos

### PR DESCRIPTION
**Pull request by MengRS**
- set up css modules
- client work

### CODE REVIEW

**Comment by MengRS** [comment]
@cjm771 for primary
@McCloudFace for secondary

File: client/index.jsx [hunk 1]
```diff
@@ -1,22 +1,53 @@
...
     )
   }
 }
 
-ReactDOM.render(<App />, document.getElementById('app'));
\ No newline at end of file
+ReactDOM.render(<App />, document.getElementById('app'));
+
+window.NT = window.NT || {};
+window.NT.TrailPhotosService = App;
```
Comment (cjm771) [inline client/index.jsx]: Make changes that we discussed, here and push should be good.

**Review by cjm771 — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -6,22 +6,25 @@
...
+    "modulePathIgnorePatterns": [
+      "helpers"
+    ]
   },
   "engines": {
     "node": ">=6.13.0"
   },
   "scripts": {
-    "start": "nodemon server/index.js & npm run react-dev | npm run openPage",
+    "start": "nodemon server/index.js | npm run openPage",
```
Comment (cjm771) [inline package.json:17]: if this is the one we should launch, take off run openpage as other people don't want a browser window.

File: client/index.jsx (hunk 1 above)
Comment (MengRS) [reply client/index.jsx]: Done

**Comment by MengRS** [comment]
@cjm771, done with the requested changes - I also changed from `bundle.js` to `app.js`.

**Review by cjm771 — APPROVED** [review]
see my only comment about your startup script other than that works great! id also not owrry about rendering the amount of recordings and stuff in your widget, we can do that in the tab (better ui than all trails)
image

**Pull request merged**